docs/resources/Guild.md

```diff
@@ -55,6 +55,8 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 
 ** \* These fields are only sent within the [GUILD_CREATE](#DOCS_TOPICS_GATEWAY/guild-create) event **
 
+** \*\* These fields are only sent when using the [GET Current User Guilds](#DOCS_RESOURCES_USER/get-current-user-guilds) endpoint and are relative to the requested user **
+
```

Comment:
@nekokatt @tpcstld Is this a bit more clear for these two fields?

Reply:
I think this is good.